### PR TITLE
tags/student/*.tag: change to 2-space indentation #7352

### LINKED ISSUE #7352: *.jsp: change to 2-space indentation
Refactor all *.jsp pages to use 2-space indentation instead of 4.

### PR DESCRIPTION
**Pull request by sujeet14108**
Fixes partially #7352
Applied to all files in `src/main/webapp/WEB-INF/tags/student/*`

### CODE REVIEW

**Comment by sujeet14108** [comment]
@whipermr5  Ready for review

**Comment by sujeet14108** [comment]
@whipermr5 please review

File: src/main/webapp/WEB-INF/tags/student/navBar.tag [hunk 1]
```diff
@@ -5,45 +5,47 @@
...
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <t:teammatesLogo/>
     </div>
+    <div class="collapse navbar-collapse" id="contentLinks">
+      <ul class="nav navbar-nav">
+        <li<c:if test="${fn:contains(data.class,'StudentHome')}"> class="active"</c:if>>
+          <a class="navLinks" id="studentHomeNavLink" href="${data.studentHomeLink}"
+            <c:if test="${isUnregistered}">data-unreg="true"</c:if>>
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/student/navBar.tag]: This is an attribute of the `a` tag; it needs more indentation.

File: src/main/webapp/WEB-INF/tags/student/navBar.tag [hunk 2]
```diff
@@ -5,45 +5,47 @@
...
+            Home
+          </a>
+        </li>
+        <li<c:if test="${fn:contains(data.class,'StudentProfilePage')}"> class="active"</c:if>>
+          <a class="navLinks" id="studentProfileNavLink" href="${data.studentProfileLink}"
+            <c:if test="${isUnregistered}">data-unreg="true"</c:if>>
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/student/navBar.tag]: Same here.

File: src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag [hunk 3]
```diff
@@ -7,159 +7,159 @@
...
-                Upload/Edit Photo
-            </button>
-        </div>
+  <h3 id="studentName">
+    <strong>${profile.name}</strong>
+  </h3>
+  <br>
+  <div class="form-group row">
+    <div class="col-xs-6 col-sm-5 col-md-3 cursor-pointer"
+       title="<%= Const.Tooltips.STUDENT_PROFILE_PICTURE %>"
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag]: This needs more indentation.

File: src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag [hunk 4]
```diff
@@ -7,159 +7,159 @@
...
+       data-toggle="tooltip"
+       data-placement="top">
+      <img id="profilePic"
+         src="${profile.pictureUrl}"
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag]: This too.

File: src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag [hunk 5]
```diff
@@ -7,159 +7,159 @@
...
+        Upload/Edit Photo
+      </button>
+    </div>
+  </div>
+  <form class="form center-block"
+      role="form"
+      method="post"
+      action="<%= Const.ActionURIs.STUDENT_PROFILE_EDIT_SAVE %>">
+    <div class="form-group"
+       title="<%= Const.Tooltips.STUDENT_PROFILE_SHORTNAME %>"
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag]: This too.

File: src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag [hunk 6]
```diff
@@ -7,159 +7,159 @@
...
+       data-toggle="tooltip"
+       data-placement="top">
+      <label for="studentNickname">
+        The name you prefer to be called by Instructors
+      </label>
+      <input id="studentShortname"
+           name="<%= Const.ParamsNames.STUDENT_SHORT_NAME %>"
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag]: Too much indentation.

File: src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag [hunk 7]
```diff
@@ -7,159 +7,159 @@
...
+           class="form-control"
+           type="text"
+           data-actual-value="<c:out value="${profile.shortName}"/>"
+           value="<c:out value="${profile.shortName}"/>"
+           placeholder="How the instructor should call you">
+    </div>
+    <div class="form-group"
+       title="<%= Const.Tooltips.STUDENT_PROFILE_EMAIL %>"
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag]: This needs more indentation.

File: src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag [hunk 8]
```diff
@@ -7,159 +7,159 @@
...
+       data-toggle="tooltip"
+       data-placement="top">
+      <label for="studentEmail">
+        Long term contact email <em class="font-weight-normal emphasis text-muted small">- only visible to your instructors</em>
+      </label>
+      <input id="studentEmail"
+           name="<%= Const.ParamsNames.STUDENT_PROFILE_EMAIL %>"
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag]: Too much indentation.

File: src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag [hunk 9]
```diff
@@ -7,159 +7,159 @@
...
+           class="form-control"
+           type="email"
+           data-actual-value="<c:out value="${profile.email}"/>"
+           value="<c:out value="${profile.email}"/>"
+           placeholder="Contact Email (for your instructors to contact you beyond graduation)">
+    </div>
+    <div class="form-group"
+       title="<%= Const.Tooltips.STUDENT_PROFILE_INSTITUTION %>"
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag]: This needs more indentation.

File: src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag [hunk 10]
```diff
@@ -7,159 +7,159 @@
...
+       data-toggle="tooltip"
+       data-placement="top">
+      <label for="studentInstitution">
+        Institution
+      </label>
+      <input id="studentInstitution"
+           name="<%= Const.ParamsNames.STUDENT_PROFILE_INSTITUTION %>"
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag]: Too much indentation.

File: src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag [hunk 11]
```diff
@@ -7,159 +7,159 @@
...
+           class="form-control"
+           type="text"
+           data-actual-value="<c:out value="${profile.institute}"/>"
+           value="<c:out value="${profile.institute}"/>"
+           placeholder="Your Institution">
+    </div>
+    <div class="form-group"
+       title="<%= Const.Tooltips.STUDENT_PROFILE_NATIONALITY %>"
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag]: This needs more indentation.

File: src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag [hunk 12]
```diff
@@ -7,159 +7,159 @@
...
+       data-toggle="tooltip"
+       data-placement="top">
+      <label for="studentNationality">
+        Nationality
+      </label>
+      <select id="studentNationality"
+          name="<%=Const.ParamsNames.STUDENT_NATIONALITY%>"
+          class="form-control" style="width: 300px">
+        <c:forEach items="${profile.nationalitySelectField}"
+               var="option">
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag]: Too much indentation; anyway combine this with the previous line.

File: src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag [hunk 13]
```diff
@@ -7,159 +7,159 @@
...
+      </c:if>
+    </div>
+    <div class="form-group">
+      <label for="studentGender">
+        Gender
+      </label>
+      <div id="studentGender">
+        <label for="genderMale" class="radio-inline">
+          <input id="genderMale"
+               name="<%= Const.ParamsNames.STUDENT_GENDER %>"
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag]: Too much indentation.

File: src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag [hunk 14]
```diff
@@ -7,159 +7,159 @@
...
+               class="radio"
+               type="radio"
+               value="<%= Const.GenderTypes.MALE %>"
+               <c:if test="${profile.gender == MALE}">checked</c:if>> Male
+        </label>
+        <label for="genderFemale" class="radio-inline">
+          <input id="genderFemale"
+               name="<%= Const.ParamsNames.STUDENT_GENDER %>"
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag]: Too much indentation.

File: src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag [hunk 15]
```diff
@@ -7,159 +7,159 @@
...
+               class="radio"
+               type="radio"
+               value="<%= Const.GenderTypes.FEMALE %>"
+               <c:if test="${profile.gender == FEMALE}">checked</c:if>> Female
+        </label>
+        <label class="radio-inline" for="genderOther">
+          <input id="genderOther"
+               name="<%= Const.ParamsNames.STUDENT_GENDER %>"
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag]: Too much indentation.

File: src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag [hunk 16]
```diff
@@ -7,159 +7,159 @@
...
+               class="radio"
+               type="radio"
+               value="<%= Const.GenderTypes.OTHER %>"
+               <c:if test="${profile.gender == OTHER}">checked</c:if>> Not Specified
+        </label>
+      </div>
+    </div>
+    <div class="form-group"
+       title="<%= Const.Tooltips.STUDENT_PROFILE_MOREINFO %>"
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag]: This needs more indentation.

File: src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag [hunk 17]
```diff
@@ -7,159 +7,159 @@
...
+       data-toggle="tooltip"
+       data-placement="top">
+      <label for="studentNationality">
+        More info about yourself
+      </label>
+      <%-- Do not add whitespace between the opening and closing tags --%>
+      <textarea id="studentMoreInfo"
+            name="<%= Const.ParamsNames.STUDENT_PROFILE_MOREINFO %>"
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag]: Too much indentation.

**Comment by sujeet14108** [comment]
Done

**Pull request merged**